packages/near-fast-auth-signer/.gitignore

```diff
@@ -25,3 +25,6 @@ yarn-error.log*
 
 # Sentry Config File
```

Comment:
Verify email it's send to a device where the user recently logged in

1 - On Firefox create an account
2 - After login, logout and click SignIn
3 - Before the email it's autofilled, paste the email copied from MinuteMail
4 - Click Login quickly
5 - You will end up on VerifyEmail page even thought the device was recently added

https://drive.google.com/file/d/1N8Pg-M77sZZGVH4Nyki19dUvmHznL3_H/view?usp=sharing

Reply:
@esaminu